code/modules/food&drinks/kitchen machinery/processor.dm

```diff
@@ -34,7 +34,7 @@
 	var/mob/living/simple_animal/slime/picked_slime
 	for(var/mob/living/simple_animal/slime/slime in range(1,src))
 		if(slime.loc == src)
```

Comment:
This shouldn't even DO anything, range() shouldn't get a slime that's literally inside the machine already.